cl/search/models.py

```diff
@@ -2731,6 +2731,11 @@ class OpinionCluster(AbstractDateTimeModel):
         storage=IncrementingAWSMediaStorage(),
         blank=True,
     )
+    harvard_id = models.CharField(
```

Comment:
Not convinced that `harvard_id` is the best name (`cap_id` is a leading alternative). Meh, this keeps it in-line with its `filepath_*_harvard` neighbors, above.

Reply:
Seems right to me.